docs/src/mpi.md

```diff
@@ -34,9 +34,16 @@ for details.
 ### 2. Using parallel HDF5 libraries
 
 As detailed in [Using custom or system provided HDF5 binaries](@ref), set the
-`JULIA_HDF5_PATH` environment variable to the path where the parallel HDF5
-binaries are located.
-Then run `]build HDF5` from Julia.
+preference `libhdf5path` to the path where the parallel HDF5 binaries are located.
+This can be done by:
+
+```julia
+julia> using Preferences, UUIDs
+
+julia> set_preferences!(
+           UUID("f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"), # UUID of HDF5.jl
+           "libhdf5path" => "/path/to/your/hdf5/binaries/", force = true)
+```
```

Comment:
Also mention here that the underlying MPI library might have to be changed (using MPIPreferences.jl)?

Reply:
Also, here and in the section on custom binaries you should probably include the command to use if you want to reset the usage to the HDF5_jll provided library.

Reply:
Regarding your first point: How to set the MPI library with MPIPreferences.jl is described in the previous section, see [here](https://juliaio.github.io/HDF5.jl/stable/mpi/#.-Using-system-provided-MPI-libraries).
Regarding your second point: You mean deleting the LocalPreferences.toml or `@delete_preferences!` or something else?

Reply:
> How to set the MPI library with MPIPreferences.jl is described in the previous section

Gotcha 👍



> You mean deleting the LocalPreferences.toml or `@delete_preferences!` or something else?

Yes. Many Julia users are not aware (yet) of how the Preferences mechanism works. For them, it might be surprising to not be able to load HDF5 anymore and no helpful error message on how to fix it. Maybe explain it here in the docs, but also consider adding it to the error messages below.

Reply:
Ok, I will do that.

Reply:
I extended the documentation with your proposals in f292a51d8404beb52d7bf62d5ece05c9414a6b43.

Reply:
Great, looks good!